src/sentry/static/sentry/app/routes.jsx

```diff
@@ -1257,194 +1259,228 @@ function routes() {
+            }
+            component={errorHandler(LazyLoad)}
+          />
+        </Route>
+
+        <Route
+          path="/organizations/:orgId/events/"
+          componentPromise={() =>
+            import(/* webpackChunkName: "EventsContainer" */ 'app/views/events')
+          }
+          component={errorHandler(LazyLoad)}
+        >
+          <IndexRoute
+            componentPromise={() =>
+              import(/* webpackChunkName: "Events" */ 'app/views/events/events')
+            }
+            component={errorHandler(LazyLoad)}
+          />
+        </Route>
+
+        <Route
+          path="/organizations/:orgId/monitors/"
+          componentPromise={() =>
+            import(/* webpackChunkName: "MonitorsContainer" */ 'app/views/monitors')
+          }
+          component={errorHandler(LazyLoad)}
+        >
+          <IndexRoute
+            componentPromise={() =>
+              import(/* webpackChunkName: "Monitors" */ 'app/views/monitors/monitors')
             }
             component={errorHandler(LazyLoad)}
           />
           <Route
-            path="all-events/"
+            path="/organizations/:orgId/monitors/create/"
             componentPromise={() =>
-              import(
-                /* webpackChunkName: "ReleaseAllEvents" */ 'app/views/releases/detail/releaseAllEvents'
-              )
+              import(/* webpackChunkName: "MonitorCreate" */ 'app/views/monitors/create')
             }
             component={errorHandler(LazyLoad)}
           />
           <Route
-            path="artifacts/"
+            path="/organizations/:orgId/monitors/:monitorId/"
             componentPromise={() =>
               import(
-                /* webpackChunkName: "ReleaseArtifacts" */ 'app/views/releases/detail/releaseArtifacts'
+                /* webpackChunkName: "MonitorDetails" */ 'app/views/monitors/details'
               )
             }
             component={errorHandler(LazyLoad)}
           />
           <Route
-            path="commits/"
+            path="/organizations/:orgId/monitors/:monitorId/edit/"
             componentPromise={() =>
-              import(
-                /* webpackChunkName: "ReleaseCommits" */ 'app/views/releases/detail/releaseCommits'
-              )
+              import(/* webpackChunkName: "MonitorEdit" */ 'app/views/monitors/edit')
             }
             component={errorHandler(LazyLoad)}
           />
         </Route>
 
         <Route
-          path="/organizations/:orgId/releases-v2/"
+          path="/organizations/:orgId/releases/"
           componentPromise={() =>
-            import(/* webpackChunkName: "ReleasesV2Container" */ 'app/views/releasesV2')
+            decideReleasesVersion(v2 =>
+              v2
+                ? import(
+                    /* webpackChunkName: "ReleasesV2Container" */ 'app/views/releasesV2'
+                  )
+                : import(/* webpackChunkName: "ReleasesContainer" */ 'app/views/releases')
```

Comment:
A nice benefit of this approach is that removing releases v1 should be safe and simple.